### PR TITLE
[plugin][captcha] - not so invisible Captcha

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue #21974 .

### Testing Instructions

see #21974

### Expected result
if no captcha enabled i can register without error msg/exception

### Actual result
if no captcha enabled error msg/exception

### CODE REVIEW

**Comment by zero-24** [comment]
Already tested by @joeforjoomla here: https://github.com/joomla/joomla-cms/issues/21974#issuecomment-418179799

**Comment by ghost** [comment]
Altered Test of @joeforjoomla<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21979">issues.joomla.org/tracker/joomla-cms/21979</a>.</sub>

**Comment by BurtNL** [comment]
I have tested this item :white_check_mark: successfully on 4334a007b503c56d0e6fdbd11ef22b61f27652f0<br /><br />Applied the patch and tested successfully on Joomla 3.9.0-alpha2-dev.
The warning about the missing ReCaptcha is not longer visible, user registration via website is now working as expected.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21979">issues.joomla.org/tracker/joomla-cms/21979</a>.</sub>

**Comment by ka3media** [comment]
I have tested this item :white_check_mark: successfully on 4334a007b503c56d0e6fdbd11ef22b61f27652f0<br /><br />Tested on Joomla! Version 3.9.0-alpha Alpha [ Amani ] 31-August-2018

Error message disappears. <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21979">issues.joomla.org/tracker/joomla-cms/21979</a>.</sub>

**Comment by ghost** [comment]
Ready to Commit after two successful tests.

File: libraries/src/Form/Field/CaptchaField.php [hunk 1]
```diff
@@ -116,6 +116,7 @@ public function setup(\SimpleXMLElement $element, $value, $group = null)
 		if ($plugin === 0 || $plugin === '0' || $plugin === '' || $plugin === null)
 		{
 			$this->hidden = true;
+			return false;
```
Comment (roland-d) [inline libraries/src/Form/Field/CaptchaField.php]: Just a little nag, we need an empty line before the return.

Comment (alikon) [reply libraries/src/Form/Field/CaptchaField.php]: added

**Pull request merged**